### DISCUSSION TITLE
Can I fork your docs  ?

### DISCUSSION THREAD

**Discussion by slashformotion** [jpanther/congo#101] [General]
I am building my own theme based on the picocss framework, I would like to use some parts of your docs and contributing guidelines. Can I do it ? *(Obviously I will mention this project)*

Not related: You have a pull request waiting to improve french translations

**Comment by jpanther** [comment]
Yes, that's fine. The documentation is [licenced](https://github.com/jpanther/congo/blob/dev/LICENSE) under the same MIT licence as the project so as long as you follow the licence terms there's no issue.
